xapi-model-spring-boot-autoconfigure/src/main/java/dev/learning/xapi/XapiModelAutoConfiguration.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2016-2023 Berry Cloud Ltd. All rights reserved.
+ */
+
+package dev.learning.xapi;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.cfg.CoercionAction;
+import com.fasterxml.jackson.databind.cfg.CoercionInputShape;
+import com.fasterxml.jackson.databind.type.LogicalType;
+
+import dev.learning.xapi.jackson.XapiStrictLocaleModule;
+import dev.learning.xapi.jackson.XapiStrictNullValuesModule;
+import dev.learning.xapi.jackson.XapiStrictTimestampModule;
+
+import org.springframework.boot.autoconfigure.AutoConfigureBefore;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.autoconfigure.jackson.Jackson2ObjectMapperBuilderCustomizer;
+import org.springframework.boot.autoconfigure.jackson.JacksonProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Auto-configure xApi model.
+ *
+ * @author István Rátkai (Selindek)
+ */
+@Configuration
+@AutoConfigureBefore(value = JacksonProperties.class)
+public class XapiModelAutoConfiguration {
+
+  @Bean
+  public Jackson2ObjectMapperBuilderCustomizer Customizer() {
+    return builder -> builder.postConfigurer(objectMapper -> {
+      objectMapper.configure(DeserializationFeature.ACCEPT_SINGLE_VALUE_AS_ARRAY, true);
+    });
+  }
+
+  @Bean
+  @ConditionalOnProperty("xApi.model.strictLocale")
```

Comment:
These conditionals should be on property value.

It is confusing to me that:

`xApi.model.strictLocale = true`

is the same as

`xApi.model.strictLocale = false`

Should probably be:

`Java
@ConditionalOnProperty(
  value="xApi.model.strictLocale", 
  havingValue = "true", 
  matchIfMissing = true)
`